.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Integration Test
+
+on:
+  push:
```

Comment:
Do we not want this to run on pull requests?

Reply:
Okay